themes/default/content/blog/changes-to-import/index.md

```diff
@@ -0,0 +1,345 @@
+---
+title: "Changes to `pulumi import`"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-02-18T10:00:59Z
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting your post for review.
+draft: true
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: Changes to the pulumi import system and terraform bridge
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - fraser-waters
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Importing resources to Pulumi is a fundamental requirement to allow bringing over existing infrastructure into Pulumi programs. Unfortunately the existing system has resulted in users hitting a number of issues that has made this task harder than it ought to be.
+
+Today we're announcing plans to rework the import system to try and fix most of these issues. This system is still not perfect and we want to give users a heads up on issues they may still need to work around.
```

Comment:
This paragraph is a little unclear. Are you saying that we have made changes to how import works, or that we are planning to make changes, or both? It could be clarified in here and also give more of a preview to readers as to what those changes are and how it benefits them.

Reply:
Well it depends when we push this vs the CLI release. I was thinking we could release this as soon as possible and then actually make the CLI changes the week after. But product team might decided it best to just release them together. A confirmation from @meagancojocar and @lukehoban on what best path to take here would be good.

Reply:
we should be releasing the blog and the update together rather than decoupled. that's the strategy for launches we set last year described in this [doc](https://docs.google.com/document/d/1dGXVjYms3pEqv58V9YSYwOmz0-OD6_toaFQhhz8lKxA/edit)

Reply:
Yeah - let's hold off on the blog post until we've delivered both the CLI feature and the resource provider updates for at least the "big 4" providers.  We can then frame this post as detailing how we've "improved the `pulumi import` experience".

Reply:
The terraform changes are already landed in tfbridge and more landed today, so their making their way out. The main engine changes are targeted for the March 9th release.